src/components/tooltip/__tests__/__snapshots__/Tooltip.test.tsx.snap

```diff
@@ -247,16 +247,9 @@ exports[`components/tooltip/Tooltip render() should render with close button if
     role="tooltip"
   >
     hi
-    <PlainButton
-      className="tooltip-close-button"
+    <CloseButton
```

Comment:
Should we add a new snapshot test for this component?

Reply:
think we're leaning away from snapshots for new tests, so I'll add some manual tests! good ide atho